packages/e2e/cypress/e2e/api/dashboard.cy.ts

```diff
@@ -163,24 +164,22 @@ describe('Lightdash dashboard', () => {
                     },
                 ).then(({ chart: newChart2, dashboard: updatedDashboard2 }) => {
                     expect(updatedDashboard2.tiles.length).to.eq(2);
-                    const firstTile = updatedDashboard2
-                        .tiles[0] as DashboardChartTile;
-                    const secondTile = updatedDashboard2
-                        .tiles[1] as DashboardChartTile;
-
-                    expect(
-                        firstTile.properties.savedChartUuid,
-                        "Check if first tile didn't change",
-                    ).to.eq(newChart.uuid);
-                    // assert second tile is correct
-                    expect(
-                        secondTile.properties.savedChartUuid,
-                        'Check if second tile is correct',
-                    ).to.eq(newChart2.uuid);
-                    expect(
-                        secondTile.properties.belongsToDashboard,
-                        'Check if second tile belongs to a dashboard',
-                    ).to.eq(true);
+                    const firstTile = updatedDashboard2.tiles.find(
+                        ({ properties }: DashboardChartTile) =>
+                            properties.savedChartUuid === newChart.uuid &&
+                            properties.belongsToDashboard,
+                    );
+                    const secondTile = updatedDashboard2.tiles.find(
+                        ({ properties }: DashboardChartTile) =>
+                            properties.savedChartUuid === newChart2.uuid &&
+                            properties.belongsToDashboard,
+                    );
+                    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
Tried to fix linting as instructed [here](https://github.com/cypress-io/eslint-plugin-cypress?tab=readme-ov-file#chai-and-no-unused-expressions) without success. Looks like one of the plugins are not maintain anymore.